examples/rust/ex1-volume/src/cli.yml

```diff
@@ -0,0 +1,52 @@
+name: Example 1 - Volume
+version: "1.0"
+author: Jeremy L Thompson <thompson.jeremy.luke@gmail.com>
+about: This example uses the mass matrix to compute the length, area, or volume of a region, depending upon runtime parameters.
+args:
+    - ceed:
+        short: -c
+        long: --ceed
+        help: Set libCEED backend resource to use
+        takes_value: true
+        default_value: /cpu/self
```

Comment:
Do you prefer this over [structopt](https://docs.rs/structopt/0.3.21/structopt/)? It lets the compiler check that you've spelled and typed your options consistently and saves the unwrap.

Reply:
I have no strong feelings either way. I picked clap only because it seems a bit more widely used. Having the command line options in a separate yaml seemed like a nice way to keep main tidy.

Reply:
Ah, `structopt` uses `clap`, but the result of parsing is a struct instead of an object you query by string. No big deal right now.

Reply:
refactoring to use structopt made testing easier